labware-library/src/labware-creator/components/TextField.tsx

```diff
@@ -46,7 +47,7 @@ export const TextField = (props: Props): React.Node => {
               caption={caption}
               placeholder={placeholder}
               onChange={makeHandleChange({ field, form })}
-              onBlur={(e: SyntheticEvent<HTMLInputElement>) => {
+              onBlur={(e: React.FormEvent<HTMLInputElement>) => {
```

Comment:
This annotation might not be needed, but if it is:

```suggestion
              onBlur={(e: React.FocusEvent<HTMLInputElement>): void => {
```